homeassistant/components/unifiprotect/config_flow.py

```diff
@@ -97,10 +98,14 @@ async def _async_get_nvr_data(
             errors[CONF_PASSWORD] = "invalid_auth"
```

Comment:
Please change the functions, eg `_async_get_nvr_data`, that are both decorated with `@callback` and defined as coroutine functions. If we don't need to await inside the function the function can be a callback. If it needs to await it should be a coroutine function. It's not both.